backend/views/index.pug

```diff
@@ -0,0 +1 @@
+!= calendlyPage
```

Comment:
curious, what's this for?

Reply:
https://expressjs.com/en/guide/using-template-engines.html

This link that Philip sent explains how the file templating works. Essentially it will replace calendlyPage with the actual JS string which holds the html and render it.

Reply:
I see but right now we aren't doing templating so do you plan to add that? I defo think that'll be great because it will make for a nicer codebase

Reply:
gotcha! if we are using a templating engine, lets make full use of it!